### PR TITLE
Add nuget tracer instructions for Docker

### PR DESCRIPTION
**Pull request by Hawxy**
### What does this PR do?
Adds instructions for using the `Datadog.Monitoring.Distribution` nuget package with docker images.

### Motivation
Without running the script, APM will not work within Linux images. I got stuck with this yesterday until I looked at the samples in the dd-trace repository.

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by zacharycmontoya — COMMENTED** [review]
Thank you for the contribution! After making sure the link in the markup works as expected, I'd say this is good to go 👍🏼

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 1]
```diff
@@ -192,7 +192,15 @@ For information about the different methods for setting environment variables, s
...
-2. For standalone applications, manually restart the application.
+2. For Docker images running on Linux, configure the image to run the `createLogPath.sh` script:
+  
+   ```
+   RUN /<APP_DIRECTORY>/datadog/createLogPath.sh
+   ```
+
+  Docker examples are available in the dd-trace-dotnet [repository][8].
+
+3. For standalone applications, manually restart the application.
```
[the remark this answers is not in the document]
Comment (kayayarai) [reply content/en/tracing/setup_overview/setup/dotnet-core.md:203]: Good catch. I'll add comments to this effect inline

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 2]
```diff
@@ -192,7 +192,15 @@ For information about the different methods for setting environment variables, s
...
    Windows x86      | `<APP_DIRECTORY>\datadog\win-x86\Datadog.Trace.ClrProfiler.Native.dll`
 
-2. For standalone applications, manually restart the application.
+2. For Docker images running on Linux, configure the image to run the `createLogPath.sh` script:
+  
+   ```
+   RUN /<APP_DIRECTORY>/datadog/createLogPath.sh
+   ```
+
+  Docker examples are available in the dd-trace-dotnet [repository][8].
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:201]:
```suggestion
  Docker examples are available in the dd-trace-dotnet [repository][1].
```

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 3]
```diff
@@ -649,3 +657,4 @@ When using `systemctl` to run .NET applications as a service, you can also set e
 [5]: /tracing/setup_overview/compatibility_requirements/dotnet-core#integrations
 [6]: /tracing/setup_overview/custom_instrumentation/dotnet/
 [7]: https://www.freedesktop.org/software/systemd/man/systemctl.html#set-environment%20VARIABLE=VALUE%E2%80%A6
+[8]: https://github.com/DataDog/dd-trace-dotnet/tree/master/tracer/samples/NugetDeployment
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:660]:
```suggestion
```

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 4]
```diff
@@ -192,7 +192,15 @@ For information about the different methods for setting environment variables, s
...
-2. For standalone applications, manually restart the application.
+2. For Docker images running on Linux, configure the image to run the `createLogPath.sh` script:
+  
+   ```
+   RUN /<APP_DIRECTORY>/datadog/createLogPath.sh
+   ```
+
+  Docker examples are available in the dd-trace-dotnet [repository][8].
+
+3. For standalone applications, manually restart the application.
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:205]:
```suggestion

[1]: https://github.com/DataDog/dd-trace-dotnet/tree/master/tracer/samples/NugetDeployment
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
Link references must go inside the tab shortcodes.

File: content/en/tracing/setup_overview/setup/dotnet-core.md [hunk 5]
```diff
@@ -192,9 +192,18 @@ For information about the different methods for setting environment variables, s
...
    Windows x86      | `<APP_DIRECTORY>\datadog\win-x86\Datadog.Trace.ClrProfiler.Native.dll`
 
-2. For standalone applications, manually restart the application.
+2. For Docker images running on Linux, configure the image to run the `createLogPath.sh` script:
+  
+   ```
+   RUN /<APP_DIRECTORY>/datadog/createLogPath.sh
+   ```
+
+  Docker examples are available in the dd-trace-dotnet [repository][1].
```
Comment (kayayarai) [inline content/en/tracing/setup_overview/setup/dotnet-core.md:201]:
```suggestion
   Docker examples are available in the [`dd-trace-dotnet` repository][1].
```

**Review by kayayarai — APPROVED** [review]

**Comment by kayayarai** [comment]
Thank you for the PR!

**Pull request merged**